src/app/page/Recipient/MissionFeedback.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useContext, useEffect } from "react";
+import { Mission } from "../../model";
+import { MissionStatus } from "../../model/schema";
+import Snackbar from "../../component/Snackbars";
+import {
+  DeliveryConfirmation,
+  Feedback,
+  FeedbackWithSuccess,
+  Acknowledgement,
+  Step,
+} from "./Steps";
+import { makeStyles, Slide, Modal, Backdrop, Paper, IconButton } from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    overflow: "auto",
+    paddingTop: "5rem",
+    paddingBottom: "2rem",
+    width: "100%",
+    maxWidth: theme.breakpoints.width("sm"),
+    position: "relative",
+    height: "100%",
+  },
+  modal: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    height: "100vh",
```

Comment:
Might want to verify on an actual mobile device, that100vh looks ok. I was styling our navigation hamburger menu and had unintentional consequences in a mobile phone browser due to the address bar actually consuming vertical space on the screen... so the content ended up being cut off.
So I ended up using `100%` instead.

Reply:
mmmm good point. ugh I hate the mobile browser experience